src/test/scala/wdl4s/SyntaxErrorSpec.scala

```diff
@@ -256,5 +256,17 @@ class SyntaxErrorSpec extends FlatSpec with Matchers {
         |}
       """.stripMargin)
   }
+  it should "detect when expressions within a command block don't reference actual variables" in {
+    expectError(
+      """task a {
+        |  Int x
+        |  command { ./script ${x+y} }
+        |}
+        |
+        |workflow w {
+        |  call a
```

Comment:
I believe `call a {input: x=5, y=5 }` ought to return an syntax error in this case.  If that syntax error is not already present I'll add it.